File: src/main/java/com/rs/java/game/area/banks/PortPhasmatysBank.java
package com.rs.java.game.area.banks;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.Shape;
import com.rs.java.game.area.shapes.Polygon;

public final class PortPhasmatysBank extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Port Phasmatys Bank";
	}

	public PortPhasmatysBank() {
		super(
				new Polygon(new WorldTile[] {
				new WorldTile(3686, 3461, 0),
				new WorldTile(3686, 3471, 0),
				new WorldTile(3699, 3471, 0),
				new WorldTile(3699, 3461, 0),
				new WorldTile(3686, 3461, 0),
		}) );
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.SAFEZONE;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/banks/PortPhasmatysBank.kt
package com.rs.kotlin.game.world.area.areas.banks

import com.rs.java.game.WorldTile
import com.rs.kotlin.game.world.area.Area
import com.rs.kotlin.game.world.area.shape.Polygon

class PortPhasmatysBank : Area(
    Polygon(
        arrayOf(
            WorldTile(3686, 3461, 0),
            WorldTile(3686, 3471, 0),
            WorldTile(3699, 3471, 0),
            WorldTile(3699, 3461, 0),
            WorldTile(3686, 3461, 0),
        )
    )
) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Port Phasmatys Bank"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.SAFEZONE
    }
}
